run_release.py

```diff
@@ -44,7 +44,7 @@
 
 WHATS_NEW_TEMPLATE = """
 ****************************
-  What's New In Python {version}
+  What's new in Python {version}
```

Comment:
Appreciate it is an exception to the rule, but *What's New* as the name of a series of articles seems better capitalised.
```suggestion
  What's New In Python {version}
```

Reply:
It's still a title and I'd prefer keeping sentence case. This also matches the frontpage: https://docs.python.org/3/